libs/execution_context/ebpf_link.c

```diff
@@ -372,6 +372,10 @@ ebpf_link_attach_program(_Inout_ ebpf_link_t* link, _Inout_ ebpf_program_t* prog
 
     // Attach the program to the link.
     ebpf_program_attach_link(program, link);
+    //
+    // add extra ref on the attached program
+    //
```

Comment:
```suggestion

    // Add an extra reference on the attached program.
```
Follow consistent comment style.